src/Apps/cudaPlanesweepMAPTk.cpp

```diff
@@ -400,9 +397,14 @@ int main(int argc, char* argv[])
             // Compute near and far range.
             double nearZ,farZ;
             std::vector<float> zs;
+            const Eigen::Matrix3d& K = it->second.getK();
+            // Assume the image size is twice the principal point
+            const int imageWidth=static_cast<int>(K(0,2) * 2);
+            const int imageHeight=static_cast<int>(K(1,2) * 2);
+            std::cout<< "Image " << it->first << " is assumed to be " << imageWidth << "x"<< imageHeight <<std::endl;
             for(unsigned int i=0; i<points.size(); i++) {
                 Eigen::Vector3d x = (it->second.getR()*points[i] + it->second.getT());
-                Eigen::Vector3d px= 1.0/x[2]*(it->second.getK()*x);
+                Eigen::Vector3d px= 1.0/x[2]*(K * x);
                 if(x[2]>0 && px[0]>0 && px[0]<imageWidth && px[1]>0 && px[1]<imageHeight)
```

Comment:
Yes, I understand.  My images are not 1920x1080, so hard coding to that size is problematic for me.  It is very common for camera calibration to fix the principal point at the center of the image, so it's like that you will get back the same 1920x1080 values for your data.

"Image size is twice the principal point" is still an assumption, but I'll argue that it is a better assumption than hard coding 1080P, and probably will give you the same results on your data.